generators/GitHubIssueGenerator.py

```diff
@@ -32,6 +32,30 @@ class GitHubIssueGenerator(AbstractGenerator.AbstractGenerator, ReportGenerator.
         f"{ra.ResultsAggregator.ignored}": ":warning:",
     }
 
+    # Dict containing tag shortname (used in metadata) to actual tag value (in GitHub)
+    tag_mappings = {
+        "sev1": "Severity 1 - Urgent",
+        "sev2": "Severity 2 - Major",
+        "sev3": "Severity 3 - Minor",
+        "p0": "blocker (P0)",
+        "p1": "Priority/P1",
+        "p2": "Priority/P2",
+        "p3": "Priority/P3"
```

Comment:
Will we be nonperforming operations on the keys to have them return uniformly, there may be squads who are returning `Sev1` instead of `sev1` and not sure how python will reconcile.

Reply:
@Kyl-Bempah I actually handle that by using `.lower()` here: https://github.com/open-cluster-management/canary-reporting/pull/13/files#diff-03ddb2f6d9e23a271c55e9d4ba7ded8e5fcc170a96e3706e7477a637f179216aR250

Reply:
Cool, hopefully they don't change labels underneath us.